src/cephadm/cephadm

```diff
@@ -1240,18 +1240,18 @@ def make_data_dir_base(fsid, uid, gid):
     return data_dir_base
 
 def make_data_dir(fsid, daemon_type, daemon_id, uid=None, gid=None):
-    # type: (str, str, Union[int, str], int, int) -> str
-    if not uid or not gid:
-        (uid, gid) = extract_uid_gid()
+    # type: (str, str, Union[int, str], Optional[int], Optional[int]) -> str
+    if uid is None or gid is None:
```

Comment:
why does this fix the issue? can you amend the commit message?

Reply:
Because the uid and gid are zero in another container and that leads to the attempt of extracting the uid/gid, which at this point does not work. When simply testing for `None` explicitly, there's no issue.

I'll adapt the commit message accordingly.